test/VoteManager.js

```diff
@@ -121,6 +121,36 @@ describe('VoteManager', function () {
         assertBNEqual(age3, age4, 'age1, age2 not equal');
       });
 
+      it('should not be able to commit if already commited in a particular epoch', async function () {
+        const epoch = await getEpoch();
+        const votes = [100, 200, 300, 400, 500, 600, 700, 800, 900];
+        const commitment1 = utils.solidityKeccak256(
+          ['uint32', 'uint48[]', 'bytes32'],
+          [epoch, votes, '0x727d5c9e6d18ed15ce7ac8d3cce6ec8a0e9c02481415c0823ea49d847ccb9ddd']
+        );
+
+        const tx = voteManager.connect(signers[3]).commit(epoch, commitment1);
+        await assertRevert(tx, 'already commited');
+      });
+
+      it('should not be able to commit if commitment is zero', async function () {
+        const epoch = await getEpoch();
+        const tx = voteManager.connect(signers[3]).commit(epoch, '0x0000000000000000000000000000000000000000000000000000000000000000');
+        await assertRevert(tx, 'Invalid commitment');
+      });
+
+      it('should not be able to commit if already commited in a particular epoch', async function () {
```

Comment:
this title is wrong I think, as that's not what tested here, and the same title exists above.
Please correct it

Reply:
I will change it I think it should be 'should not be able to commit if staker does not exist'